app/mailers/activity_mailer.rb

```diff
@@ -30,4 +32,9 @@ def mail(options)
 
     super
   end
+
+  private
+  def subject_path_from(view_path)
```

Comment:
@custode Can you document the format of the returned translation key, otherwise it's hard to figure out where should I put the keys.